numalogic/registry/mlflow_registry.py

```diff
@@ -271,3 +267,15 @@ def __delete_stale_models(self, skeys: Sequence[str], dkeys: Sequence[str]):
             for stale_model in models_to_delete:
                 self.delete(skeys=skeys, dkeys=dkeys, version=stale_model.version)
                 _LOGGER.debug("Deleted stale model version : %s", stale_model.version)
+
+    def __load_artifacts(
+        self, skeys: Sequence[str], dkeys: Sequence[str], version_info: ModelVersion
+    ) -> Tuple[Artifact, Dict[str, Any]]:
+        model_key = self.construct_key(skeys, dkeys)
+        model = self.handler.load_model(model_uri=f"models:/{model_key}/{version_info.version}")
+        _LOGGER.info("Successfully loaded model %s from Mlflow", model_key)
+
+        run_info = mlflow.get_run(version_info.run_id)
+        metadata = run_info.data.params or None
```

Comment:
Why are setting this None?
Let's maintain consistency in datatypes. Assign an empty dict instead.

Reply:
I kept it None because we might be able to use it for more things and just not params. The typing supports None beforehand. Let's say a classification report having type as scikit object.